h2o-bindings/bin/custom/R/gen_pipeline.py

```diff
@@ -0,0 +1,28 @@
+
+extensions = dict(
+    required_params=[],
+    frame_params=[],
+    validate_required_params="",
+    set_required_params="",
+    module="""
+.h2o.fill_pipeline<- function(model, parameters, allparams) {
+  if (!is.null(model$estimator)) {
+    model$estimator_model <- h2o.getModel(model$estimator$name)
+  } else {
+    model$estimator_model <- NULL
+  }
+  model$transformers <- unlist(lapply(model$transformers, function(dt) new("H2ODataTransformer", id=dt$id, description=dt$description)))
+  # class(model) <- "H2OPipeline"
```

Comment:
good point! right I forgot about the multiple inheritance. I think I commented it out because it didn't seem necessary (single algo don't have dedicated class) and it broke some behaviour somewhere (can't remember what exactly).
The funny part is that the class is defined as follow:
```R
setClass("H2OPipeline", contains="H2OModel",
```
but afair, it was still not recognized as a model somewhere…
I will give a try to your suggestion, and see if it breaks some R tests.